src/Setup.php

```diff
@@ -105,8 +86,8 @@ public function bcgov_block_theme(): void {
      */
     public function set_javascript_variables() {
 		$javascript_variables = array(
-			'domain'    => home_url(),
-			'site-name' => 'bcgov',
+			'domain'   => home_url(),
+			'siteName' => 'cleanbc',
```

Comment:
not liking this being changed to cleanbc

Reply:
It's not, that's a change that's coming out and will remain as bcgov in Setup.php